docs/hydra/5min-tutorial.md

```diff
@@ -6,10 +6,10 @@ title: 5 Minute Tutorial
 To get started quickly, we provide a Docker Compose based example for setting up ORY Hydra, a PostgreSQL instance
 and an exemplary User Login & Consent App. You need to have the latest Docker as well as Docker Compose version installed.
 
-<img src="../../images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
+<img src="/website/static/images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
```

Comment:
Almost! :D The right url is: `src="/images/docs/hydra/oauth2-flow.gif"`, see:

* https://www.ory.sh/images/docs/hydra/oauth2-flow.gif

Reply:
Hang on! Let me try jumping off a cliff first!

On a side note, I was a victim of markdown preview on GitHub. Something should be done about this.

Reply:
Oh I see - yeah the build pipeline is a bit different and doesn't really reflect the repository structure. I'm not sure if we can actually fix that :/

Reply:
@aeneasr How about now?

Reply:
👍 thank you for updating this :)

Reply:
@aeneasr Least I can do!